specs/protocol/block_header.md

```diff
@@ -0,0 +1,13 @@
+# Block Header
+
+This document specifies the on-chain block header format.
+
+## BlockHeader
+
+| name        | type      | description                                                                       |
+|-------------|-----------|-----------------------------------------------------------------------------------|
+| `height`    | `uint32`  | Fuel block height.                                                                |
+| `number`    | `uint32`  | Ethereum block number.                                                            |
```

Comment:
Should we also specify the ethereum block hash to avoid the risk of a fork?

Reply:
Depends. L1 block hash isn't ideal, because Ethereum only lets you see the previous 256. It might be better to instead do something like some tip of deposits/valset updates commitment?